### PR TITLE
Nuri/Update: Update secure account on responsible page

### PR DESCRIPTION
**Pull request by njazuli-deriv**
Changes:

-   Update secure account on responsible page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [X] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/responsible/_securing-account.js [hunk 1]
```diff
@@ -1,55 +1,78 @@
...
+    @media ${device.tablet} {
+        padding-top: 40px;
+    }
+`
+const StyledContainer = styled(Container)`
+    @media ${device.tablet} {
+        width: 100%;
+        padding: 0 16px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/responsible/_securing-account.js:22]: Let's use the responsive props for adding these styles.

File: src/pages/responsible/_securing-account.js [hunk 2]
```diff
@@ -1,55 +1,78 @@
...
+            max-width: 100%;
         }
     }
 `
 const StyledHeader = styled(Header)`
     @media ${device.tablet} {
         font-size: var(--text-size-m);
+        margin-bottom: 24px;
     }
 `
```
Comment (sean-binary) [inline src/pages/responsible/_securing-account.js:70]: Let's do it for this as well.

File: src/pages/responsible/_securing-account.js (hunk 1 above)
Comment (njazuli-deriv) [reply src/pages/responsible/_securing-account.js:22]: I've tried it before,but still not working..

File: src/pages/responsible/_securing-account.js (hunk 2 above)
Comment (njazuli-deriv) [reply src/pages/responsible/_securing-account.js:70]: Same here..

File: src/pages/responsible/_securing-account.js (hunk 1 above)
Comment (njazuli-deriv) [reply src/pages/responsible/_securing-account.js]: fixed in latest commit

File: src/pages/responsible/_securing-account.js (hunk 2 above)
Comment (njazuli-deriv) [reply src/pages/responsible/_securing-account.js]: fixed in latest commit

**Review by sean-binary — APPROVED** [review]

**Review by suthesan-deriv — APPROVED** [review]

**Pull request merged**